legend-engine-protocol-pure/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/data/ModelStoreData.java

```diff
@@ -14,11 +14,79 @@
 
 package org.finos.legend.engine.protocol.pure.v1.model.data;
 
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.eclipse.collections.api.factory.Lists;
+import org.finos.legend.engine.protocol.pure.v1.PureProtocolObjectMapperFactory;
+import org.finos.legend.engine.protocol.pure.v1.model.valueSpecification.ValueSpecification;
+
+import java.io.IOException;
+import java.util.Iterator;
 import java.util.List;
+import java.util.Map;
 
+@JsonDeserialize(using = ModelStoreData.ModelStoreDeserializer.class)
 public class ModelStoreData extends EmbeddedData
 {
 
+    private static ObjectMapper objectMapper = PureProtocolObjectMapperFactory.getNewObjectMapper();
```

Comment:
ModelStoreData is a protocol class, right? We should move object mapper to nested class rather